certificate/views.py

```diff
@@ -1709,6 +1715,275 @@ def create_scipy_certificate_2015(certificate_path, name, qrcode, type, paper, w
         error = True
     return [None, error]
 
+@csrf_exempt
+def scipy_feedback_2016(request):
```

Comment:
Why do you need this scipy_feedback_2016 function? What is the purpose?
I don't see this function do anything!
Kindly explain.